package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "action-tower-launch",
+  "version": "3.0.0",
+  "description": "Launch Nextflow workflows on Seqera Platform via JavaScript action",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "ncc build src/index.js -o dist --source-map --license licenses.txt",
+    "test": "vitest run",
+    "test:watch": "vitest",
+    "test:coverage": "vitest run --coverage",
+    "test:ci": "vitest run --coverage",
+    "lint": "eslint src/**/*.js",
+    "package": "npm run build && npm test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/seqeralabs/action-tower-launch.git"
+  },
+  "keywords": [
+    "github-action",
+    "nextflow",
+    "seqera",
+    "tower",
+    "workflow",
+    "bioinformatics"
+  ],
+  "author": "Phil Ewels",
```

Comment:
I'm not really sure that I'm the primary author any more after this, but whatever.. 😅